types/src/on_chain_config/timed_features.rs

```diff
@@ -69,6 +69,10 @@ impl TimedFeatureFlag {
         use TimedFeatureFlag::*;
 
         match (self, chain_id) {
+            (_, MOVEMAINNET | MOVETESTNET) => Los_Angeles
+                .with_ymd_and_hms(2025, 8, 11, 17, 0, 0)
+                .unwrap()
+                .with_timezone(&Utc),
```

Comment:
Any reason why this was chosen. We can update this date to be some later time in the future I suppose without a negative side effect?